dali/dfuplus/main.cpp

```diff
@@ -107,6 +107,7 @@ void handleSyntax()
     out.append("        diffkeysrc=<old-key-name>   -- use keydiff/keypatch (src old name)\n");
     out.append("        diffkeydst=<old-key-name>   -- use keydiff/keypatch (dst old name)\n");
     out.append("        multicopy=0|1   -- each destination part gets whole file\n");
+    out.append("        preservecompression=0|1 -- optional, default is 0 (don't preserve compression)\n");
```

Comment:
I'm not convinced that maintaining compatibility (i.e. retaining the erroneous behaviour by default) is more important here than making sure that normal usage of this function behaves as people would expect (so long as we are going into a major release - we an add a note to the redbook).

@ghalliday what do you think?